imap_processing/swe/l1a/swe_l1a.py

```diff
@@ -1,10 +1,18 @@
 import logging
+from pathlib import Path
 
-from imap_processing.swe.l0 import decom_swe
+from imap_processing.swe import __version__
 from imap_processing.swe.l1a.swe_science import swe_science
+from imap_processing.swe.utils.swe_utils import (
+    SWEAPID,
+    create_dataset,
+    get_descriptor,
+)
+from imap_processing.utils import group_by_apid, sort_by_time
+from imap_processing.write_to_cdf import write_to_cdf
```

Comment:
Can you change this to import from `imap_processing.cdf_utils` now that it is in? Then you could remove this portion and avoid the duplication now instead of having to fix it later.

Reply:
I started working on it but it introduced more issues. I will revisit this on Monday.

Reply:
I think I worked out most of issue that day and forgot to update here(because I was sick) but I will double check when I rebase with new changes from Maxine's.